gateway-api/api/nearby-spots.js

```diff
@@ -0,0 +1,31 @@
+//nearby-spots.js
+
+module.exports = function(db) {
+  return function(req, res) {
+    var superagent = require('superagent');
+
+    superagent
+    .get('http://localhost:3000/spots')
```

Comment:
syntax is bad. Missing .end((err, response) => {/*All your other code goes here*/})